src/redux/books/bookSlice.js

```diff
@@ -0,0 +1,30 @@
+import { createSlice } from '@reduxjs/toolkit';
+import bookItems from '../../bookItems';
+
+const initialState = {
+  bookItems,
+  total: 0,
+};
```

Comment:
- [ ] Nice work defining a `slice` of state for `books` :clap: However the project requirement stated that the books state should be initialized with a state value `books` that contains an empty array(see image) Therefore I suggest you set the initialState with an empty array instead of import it to follow the project requirement
![Screenshot from 2023-07-17 09-40-51](https://github.com/kamranasif506/book-store/assets/91142432/c27c27b6-7c85-4cd5-8c62-1f4be0cc87c5)
**HIT: Your constant `initialState` could look like this :point_down:**
```js
const initialState = {
  bookItems: [],
  total: 0,
};
```